lightning/src/blinded_path/payment.rs

```diff
@@ -239,6 +239,19 @@ impl BlindedPaymentPath {
 				self.inner_path.blinded_hops.remove(0);
 				Ok(())
 			},
+			Ok((BlindedPaymentTlvs::Dummy, control_tlvs_ss)) => {
+				let next_node_id = node_signer.get_node_id(Recipient::Node)?;
+				let mut new_blinding_point = onion_utils::next_hop_pubkey(
+					secp_ctx,
+					self.inner_path.blinding_point,
+					control_tlvs_ss.as_ref(),
+				)
+				.map_err(|_| ())?;
+				mem::swap(&mut self.inner_path.blinding_point, &mut new_blinding_point);
+				self.inner_path.introduction_node = IntroductionNode::NodeId(next_node_id);
+				self.inner_path.blinded_hops.remove(0);
+				Ok(())
+			},
```

Comment:
Can DRY this up with the previous arm by having an inner `match` on the `BlindedPaymentTlvs` evaluating to the `next_node_id`.